build.gradle

```diff
@@ -124,3 +124,10 @@ project(':kafka') {
 		compile project(':core')
 	}
 }
+
+//JDBC
+project(':jdbc') {
```

Comment:
Is this a JDBC **Source** or a **Sink** or a **CDC** - Change Data Capture?
Maybe clarify / or start with 2 (source & sink)

Reply:
well initially just a simple source and sink